src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/client/ProfessionalApiClient.java

```diff
@@ -255,7 +255,6 @@ public  NewUserCreationRequest createNewUserRequest(String email) {
     public Map<String, Object> addNewUserToAnOrganisation(String orgId, String role, NewUserCreationRequest newUserCreationRequest, HttpStatus expectedStatus) {
         Response response = getMultipleAuthHeadersInternal()
                 .body(newUserCreationRequest)
-                .proxy("proxyout.reform.hmcts.net", 8080)
                 .post("/refdata/internal/v1/organisations/" + orgId + "/users/")

```

Comment:
I don't see any functional test case for this new functionality

Reply:
external user scenario we cannot invite user  ..knows issue with SIDAM.

Reply:
This means our External endpoint for inviting user doesn't work ? Also we just need to check in functional test case  if user is pending in UP then 403 should appear , don't need to invite user.

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/492/commits/d7c2a1899b49e00c9dbced1e4e8821b9a91295c5